website/source/docs/job-specification/job.html.md

```diff
@@ -87,8 +89,16 @@ job "docs" {
 - `update` <code>([Update][update]: nil)</code> - Specifies the task's update
   strategy. When omitted, rolling updates are disabled.
 
+- `vault_token` `(string: "")` - Specifies the Vault token that proves the
```

Comment:
Need to make it clear that it shouldn't really ever be set by the user unless hitting the JSON API